recipes/uwsgi/bld.bat

```diff
@@ -0,0 +1,6 @@
+set UWSGI_PROFILE=%RECIPE_DIR%/uwsgi_config.ini
+set UWSGI_INCLUDES=%PREFIX%/include,%PREFIX%/include/openssl
+set LDFLAGS=-L%PREFIX%/lib
+
+python setup.py install

```

Comment:
Any reason I shouldn't just used the `--old-and-unmanageable` flag? 😜
I'm a big fan of the name.